less/mixins.less

```diff
@@ -1 +1,19 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.custom-btn(@width:200px, @bgColor:@eastern-blue, @fontColor:white){
```

Comment:
You are owning these parametric mixins, they will help you understand JS Functions